tools/ms2snoop/MS2snoop.xml

```diff
@@ -0,0 +1,142 @@
+<tool id="MS2snoop" name="MS2 Snoop" version="1.0.0">
+    <description>
+        Extraction des spectres MS1 et adduits associées + Extraction + 
+        nettoyage automatisé des spectres MS2 Extraction des métadonnées
+        des fichiers RAW Export en fichier idéalement en JSON
+    </description>
+
+    <requirements>
+        <requirement type="package">r-base</requirement>
+        <requirement type="package">r-optparse</requirement>
+    </requirements>
+
+    <command><![CDATA[
+Rscript '$__tool_directory__/MS2snoop.R'
+    -o $frag_result_txt
+    -c $compound_txt
+    -f $peaklist_frag_tsv
+    -p $peaklist_preco_tsv
+    --tolmz $tolmz
+    --tolrt $tolrt
+    --seuil_ra $seuil_ra
+    --mzdecimal $mzdecimal
+    --r_threshold $r_threshold
+    --min_number_scan $min_number_scan
+;
+    ]]></command>
+
+    <inputs>
+        <param type="data" name="compound_txt" label="compounds_pos.txt" />
+        <param type="data" name="peaklist_frag_tsv" label="peaklist_fragments.tsv" />
+        <param type="data" name="peaklist_preco_tsv" label="peaklist_precursors.tsv" />
+        <param name="tolmz" type="float" value="0.01" label="MZ Tolerence" />
```

Comment:
Would it make sense to add min or max attributes to the float and integer parameters?

Reply:
I added min values, but for max values, I'm not sure at all.